Source/Engine/Physics/Colliders/CharacterController.cpp

```diff
@@ -78,6 +81,9 @@ void CharacterController::SetStepOffset(float value)
 
     _stepOffset = value;
 
+    if (GetHeight() < value)
+        SetHeight(value);
```

Comment:
Wait, why would modifying the step offset change the *height*?

Reply:
Because step offset can never be higher than height, if they would be, PhysX throws error, engine asserts that and you get a crash. If you managed to save before that you can easily get soft-locked and need to manually edit the scene file to resolve the issue.

Reply:
Dunno, maybe this is just personal preference but I'd almost expect the step offset to get clamped to the height.

Reply:
That was the way it inicially was, but on the other hand, you wouldnt know why the value doesnt change, this way you see what changed because of that.

Reply:
It can be specified in the tooltip.

Reply:
it is, but tooltip wont fix soft-lock in case this is invalid and you save the scene